data/demo/data/costs.csv

```diff
@@ -1,109 +1,89 @@
-need,setting,cost
-CI,EO,4577.3
-CI,EYS,10181.82
-CI,FEC,11449.48
-CI,IMS,27870.97
-CI,ISS,51985.15
-CI,ISSR,127442.71
-CI,MAP,12678.11
-CI,MMS,6690.84
-CI,MMSIB,8476.39
-CI,MMSOB,8038.51
-CI,MSSIB,21560.98
-CI,MSSOB,23425.6
-CI,MSSR,0
-CI,MU,4472.12
-CI,MUOB,0
-CI,NMSS,38393.8
-CI,NMSSR,0
-CI,OOE,0
-CL,EO,23125
-CL,EYS,10181.82
-CL,FEC,10727.36
-CL,IMS,23976.84
-CL,ISS,47945.72
-CL,ISSR,117000
-CL,MAP,10653.36
-CL,MMS,5383.5
-CL,MMSIB,8744.79
-CL,MMSOB,9760.7
-CL,MSSIB,20370.76
-CL,MSSOB,19434.42
-CL,MSSR,0
-CL,MU,7141
-CL,MUOB,0
-CL,NMSS,29852.37
-CL,NMSSR,0
-CL,OOE,0
-OTH,EO,0
-OTH,EYS,0
-OTH,FEC,11686.06
-OTH,IMS,34457.62
-OTH,ISS,52755.39
-OTH,ISSR,0
-OTH,MAP,19344.98
-OTH,MMS,0
-OTH,MMSIB,10000.56
-OTH,MMSOB,7136.84
-OTH,MSSIB,21965.32
-OTH,MSSOB,0
-OTH,MSSR,0
-OTH,MU,6953.21
-OTH,MUOB,0
-OTH,NMSS,15250
-OTH,NMSSR,0
-OTH,OOE,0
-SEMH,EO,10725
-SEMH,EYS,0
-SEMH,FEC,12162.44
-SEMH,IMS,42000
-SEMH,ISS,49131.87
-SEMH,ISSR,35620
-SEMH,MAP,10601.56
-SEMH,MMS,0
-SEMH,MMSIB,8701.45
-SEMH,MMSOB,5520.1
-SEMH,MSSIB,17739.44
-SEMH,MSSOB,17892.4
-SEMH,MSSR,55356.19
-SEMH,MU,8654.05
-SEMH,MUOB,0
-SEMH,NMSS,19693.33
-SEMH,NMSSR,0
-SEMH,OOE,0
-SP,EO,0
-SP,EYS,10181.82
-SP,FEC,13100.13
-SP,IMS,11269.09
-SP,ISS,0
-SP,ISSR,0
-SP,MAP,0
-SP,MMS,2748.27
-SP,MMSIB,9890.92
-SP,MMSOB,14017.94
-SP,MSSIB,19992.89
-SP,MSSOB,25864.02
-SP,MSSR,0
-SP,MU,8217.34
-SP,MUOB,5684.46
-SP,NMSS,51919.17
-SP,NMSSR,43000
-SP,OOE,0
-UKN,EO,0
-UKN,EYS,0
-UKN,FEC,6231.75
-UKN,IMS,0
-UKN,ISS,0
-UKN,ISSR,0
-UKN,MAP,14191.36
-UKN,MMS,0
-UKN,MMSIB,11144.11
-UKN,MMSOB,0
-UKN,MSSIB,24064.01
-UKN,MSSOB,0
-UKN,MSSR,0
-UKN,MU,0
-UKN,MUOB,0
-UKN,NMSS,0
-UKN,NMSSR,0
-UKN,OOE,0
\ No newline at end of file
+need,setting,cost
+U,G,1000.0
```

Comment:
Does it make sense to at least introduce some variation in cost?

Reply:
would be fairly easy to adjust a few at random...what are you thinking would be the advantage with it though?